internal/generator/vector/output/otlp/transform.go

```diff
@@ -66,7 +66,8 @@ if exists(.openshift.labels) {for_each(object!(.openshift.labels)) -> |key,value
 r.attributes = append(r.attributes,
     [{"key": "k8s.pod.uid", "value": {"stringValue": get!(.,["kubernetes","pod_id"])}},
     {"key": "k8s.container.id", "value": {"stringValue": get!(.,["kubernetes","container_id"])}},
-    {"key": "k8s.node.name", "value": {"stringValue": .hostname}}]
+    {"key": "k8s.node.name", "value": {"stringValue": .hostname}},
+    {"key": "log.iostream", "value": {"stringValue": get!(.,["kubernetes","container_iostream"])}}]
```

Comment:
This should be added to the attributes for "logdata" of containers and not the "resource" attribute.

Reply:
r.attributes is the logRecord attribute.   This line is correct.

Reply:
One more question:   
If you are setting the value to `.kubernetes.container_iostream`   then it may seem more accurate to map this to `k8s.container.iostream`  ??    or at least maybe not `openshift.log`, and instead `k8s.log`  ??

Reply:
> r.attributes is the logRecord attribute.
Good catch

@vparfonov `log.iostream` is documented in the conventions.  Please leave as-is for now